recipes/tassel/build.sh

```diff
@@ -0,0 +1,27 @@
+BINARY_HOME=$PREFIX/bin
+TASSEL_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+# Copy source to bin
+mkdir -p $BINARY_HOME
+mkdir -p $TASSEL_HOME
+cp -R $SRC_DIR/* $TASSEL_HOME/
+
+# Fix shabang lines
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $TASSEL_HOME/*.pl
+
+# Create wrapper script in bin to call TASSEL's launch scripts
+cat >"${BINARY_HOME}/run_pipeline.sh" <<EOF
+exec $CONDA_PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/run_pipeline.pl "\$@"
+EOF
+
+cat >"${BINARY_HOME}/run_anything.sh" <<EOF
+exec $CONDA_PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/run_anything.pl "\$@"
```

Comment:
Can we use a symlink here instead?

Reply:
Unfortunately no.  If I created symbolic links for TASSEL's launch wrappers, they would fail to construct Java's CLASSPATH and to find the jar file correctly when they are invoked via the symbolic links.  This is because they would use the file location of the symbolic links rather than the file location of the actual wrappers as the base path to find other libraries.

Reply:
Actually, I don't really like my workaround either.  I will crate patches for TASSEL's launch wrappers and make them work when they are called via symbolic links.